src/assets/manifest.webmanifest

```diff
@@ -0,0 +1,25 @@
+{
+  "lang": "en",
+  "dir": "ltr",
+  "name": "Mozilla Hubs",
+  "description": "A new way to get together online. Laugh, play, get stuff done, or just hang out. A WebVR experiment by Mozilla Mixed Reality.",
+  "short_name": "Hubs",
+  "icons": [
+    {
+      "src": "https://assets-prod.reticulum.io/assets/images/icon.png",
```

Comment:
Ideally this would be digested, so it will get busted out of CDN if we change it.

Reply:
(Since this is a slowly changing asset, and it might be quite complex to convince webpack to inject it properly here, I'm not opposed to us punting and just manually busting the CDN if it changes. Worth doing if its easy though.)

Reply:
most definitely. I'm going to try [`webpack-pwa-manifest`](https://github.com/arthurbergmz/webpack-pwa-manifest#webpack-pwa-manifest), so I don't reinvent the wheel with this.